### DISCUSSION TITLE
Dependency between product attributes

### DISCUSSION THREAD

**Discussion by Osamaas04** [breezefront/community#262] [1. Help]
Hello, I'm using Breeze Evolution theme and I want to add JS function tag to Player Name option in Make It Your Own attribute to hide/show Player Name text field once the customer clicks on the option also for Custom Name, so I want to know the file that contains product attributes code either phtml or xml. Thank you.
![image](https://github.com/breezefront/community/assets/130319891/5852de5f-4121-4a3e-8fda-7bc906976e0f)

**Comment by vovayatsyuk** [comment]
You don't need to modify the template to achieve what you want.

1. Create [custom.js file](https://breezefront.com/docs/custom-javascript#theme-customization-script)
2. Use the following code as the base for your logic:

    ```js
    function updateCustomNameVisibility() {
        $('.write-selector-for-custom-name-field-wrapper').toggle(
            $('.write-selector-for-player-name-combobox').val() === 'write-value-of-custom-player-name-option'
        );
    }

    $(document).on('change', '.write-selector-for-player-name-combobox', updateCustomNameVisibility);
    $(document).on('breeze:load', updateCustomNameVisibility);
    ```

**Reply by Osamaas04** [reply]
I tried this approach before the option doesn't response, so I have to get access to the attributes file. Thank you.

**Reply by vovayatsyuk** [reply]
1. Breeze does not change this template. It's taken from magento core. I recommend using template hints option to find it:
    
    ```
    bin/magento dev:template-hints:enable 
    ```

2. I still suggest using the js code I provide. There is no good reason to modify the template for you.
